libra_swarm/src/main.rs

```diff
@@ -31,12 +31,14 @@ struct Args {
     /// If specified, load faucet key from this file. Otherwise generate new keypair file.
     #[structopt(short = "f", long = "faucet_key_path")]
     pub faucet_key_path: Option<String>,
+    /// If set, starts a full node swarm connected to the first node in the validator swarm..
+    #[structopt(short = "w", long = "with_full_node_swarm")]
+    pub with_full_nodes: bool,
```

Comment:
can we change API a bit? Instead of boolean, can we specify number of full nodes. Now it's getting duplicated from nodes parameter. If I run cluster of 4, creating 4 full nodes by default feels a bit unnecessary